src/screens/send/components/enter-amount/enter-amount.tsx

```diff
@@ -46,6 +46,10 @@ export const EnterAmountComponent = (
         onChange(addedAmount.toFixed());
     };
 
+    const verifyInsuficientFunds = () => {
+        if (new BigNumber(availableAmount).eq(0)) return true;
```

Comment:
is this necessary ? if yes it seems we have a bug when we calculate the amounts and compute `insufficientFunds`

Reply:
insuficientFunds calculation is done only on input. I can make it to fire on componentDidMount or the current implementation